groovy/src/main/java/org/apache/zeppelin/groovy/GObject.java

```diff
@@ -41,308 +43,328 @@
 
 /**
  * Groovy interpreter for Zeppelin.
+ * @author dlukyanov@ukr.net / dmitry lukyanov
```

Comment:
@dlukyanov Zeppelin generally don't write author name in the source code. Even if it does, all ppl still can track and see what you contribute to Zeppelin :)